samtools/1.15/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:focal as app
+
+ARG SAMTOOLSVER=1.15
+
+LABEL base.image="ubuntu:focal"
+LABEL dockerfile.version="1"
+LABEL software="Samtools"
+LABEL software.version="${SAMTOOLSVER}"
+LABEL description="Tools (written in C using htslib) for manipulating next-generation sequencing data"
+LABEL website="https://github.com/samtools/samtools"
+LABEL license="https://github.com/samtools/samtools/blob/develop/LICENSE"
+LABEL maintainer="Erin Young"
+LABEL maintainer.email="eriny@utah.gov"
+
+ENV DEBIAN_FRONTEND noninteractive
```

Comment:
Additionally it will have to include an equals sign, so `ARG DEBIAN_FRONTEND=noninteractive`

Not sure why, but it didn't work without the equals sign